awx/ui_next/src/screens/Template/TemplateList/TemplateList.jsx

```diff
@@ -130,8 +146,14 @@ class TemplatesList extends Component {
       templates,
       itemCount,
       selected,
+      isAddOpen
     } = this.state;
-    const { match, i18n } = this.props;
+    const {
+      match,
+      i18n,
+    } = this.props;
+
+    const canAdd = true;
```

Comment:
I think we want to have a check here to determine if the user has proper RBAC to be able to add, instead of allowing all users to add Job Templates.  To do this we need to make an options request to the api to check for `actions.Post`